### ISSUE TITLE
Show only enable|disable based on state

### ISSUE THREAD

**Issue by lidel** [olizilla/dweb.link#4]
I think we should be able to test registration status somehow, perhaps via `iframe`? (see [`testProtocol` here](https://blog.mozilla.org/webdev/2010/07/26/registerprotocolhandler-enhancing-the-federated-web/))

**Comment by olizilla** [comment]
Agree! The site has to be able to check if you already have a protocol handler for ipfs:// so it can say something sensible to the user.

**Comment by olizilla** [comment]
Firefox behaves oddly with an ipfs:// scheme iframe src, opening it in a new window. https://github.com/olizilla/dweb.link/issues/7
